css/css-color/parsing/opacity-valid.html

```diff
@@ -20,14 +20,28 @@
 test_valid_value("opacity", "-100%", "-1");
 test_valid_value("opacity", "50%", "0.5");
 test_valid_value("opacity", "300%", "3");
-test_valid_value("opacity", "clamp(50%, 0%, 70%)", "calc(0.5)");
-test_valid_value("opacity", "clamp(50%, 80%, 70%)", "calc(0.7)");
-test_valid_value("opacity", "clamp(50%, 60%, 70%)", "calc(0.6)");
-test_valid_value("opacity", "min(50%, 0%)", "calc(0)");
-test_valid_value("opacity", "min(0%, 50%)", "calc(0)");
-test_valid_value("opacity", "max(50%, 0%)", "calc(0.5)");
-test_valid_value("opacity", "max(0%, 50%)", "calc(0.5)");
-test_valid_value("opacity", "min(-40%, 50%)", "calc(-0.4)");
+test_valid_value("opacity", "calc(-50% - 50%)", "calc(-100%)");
+test_valid_value("opacity", "calc(25% * 2)", "calc(50%)");
+test_valid_value("opacity", "calc(100% + 100%)", "calc(200%)");
```

Comment:
Hi @nt1m and @weinig - Firefox and Safari both fail these 3 tests (along with 8 other subtests here), in a way that looks ~fine (?) -- we're serializing as `calc(-1)` instead of `calc(-100%`) and similar for the others.

Our serialization seems fine to me, so I'm not sure that the test's expectation makes sense... Any chance you could loosen the restrictions on these to allow either formulation? Or, is the test correct to be strict in this way?